charts/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -258,6 +258,17 @@ spec:
       {{- if $.Values.priorityClassName }}
       priorityClassName: {{ $.Values.priorityClassName }}
       {{- end }}
+      {{- if .Values.global.fargate }}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: eks.amazonaws.com/compute-type
```

Comment:
Rather than hardcoding it here, perhaps we can have some syntax sugar for exclude rules for DaemonSet? Or `nodeAffinity` even as a variable + `global.faraget` sugar?

Reply:
Doesn't sound bad, let me give it a try